README.md

```diff
@@ -134,9 +134,7 @@ apt-get install                           \
 This project strives to be as normal and simple a CMake project as possible. This build workflow in particular will work, producing a static `libbeman.exemplar.a` library, ready to package with its headers:
 
 ```shell
-cmake -B build -S . -DCMAKE_CXX_STANDARD=20
-cmake --build build
-ctest --test-dir build
+cmake --workflow --preset gcc-debug
 cmake --install build --prefix /opt/beman.exemplar
```

Comment:
I suggest also including `gcc-release` since that's most likely what folks will want to install. I also had to change the install command to point to the right build directory.

```suggestion
cmake --workflow --preset gcc-release
cmake --install build/gcc-release --prefix /opt/beman.exemplar
```